src/main/drivers/resource.c

```diff
@@ -22,7 +22,10 @@
 
 #include "resource.h"
 
-const char * const ownerNames[OWNER_TOTAL_COUNT] = {
+const resourceOwner_t resourceOwnerInvalid = { .owner = OWNER_INVALID, .index = 0 };
+const resourceOwner_t resourceOwnerFree    = { .owner = OWNER_FREE,    .index = 0 };
+
+static const char * const ownerNames[OWNER_TOTAL_COUNT] = {
```

Comment:
```suggestion
static const char * const ownerNames[] = {
```
And then STATIC_ASSERT that size does match